crates/driver/src/print_build.rs

```diff
@@ -571,6 +575,89 @@ where
 
         t.entries.push(target);
     }
+
+    fn apply_secondary_rules(
+        target_entries: &mut TargetEntries,
+        module_config: &ModuleConfig,
+        parent_target_name: &str,
+        parent_include_src: &Vec<String>,
+        parent_deps: &Vec<String>,
+    ) -> Result<()> {
```

Comment:
I'm not seeing how this method can ever fail. Why does it return anything but `()`?

Reply:
Addressed in https://github.com/bazeltools/bzl-gen-build/commit/c97d4b7c8d900f53d63d9e54182d7ffbc0fc7945 by returning Unit.